titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -500,29 +509,34 @@ public static Day toCoreDay(com.netflix.titus.grpc.protogen.Day grpcDay) {
         }
     }
 
-    public static List<HourlyTimeWindow> toCoreHourlyTimeWindows(List<com.netflix.titus.grpc.protogen.TimeWindow.HourlyTimeWindow> grpcHourlyTimeWindows) {
+    public static List<HourlyTimeWindow> toCoreHourlyTimeWindows
+            (List<com.netflix.titus.grpc.protogen.TimeWindow.HourlyTimeWindow> grpcHourlyTimeWindows) {
         return grpcHourlyTimeWindows.stream().map(GrpcJobManagementModelConverters::toCoreHourlyTimeWindow).collect(Collectors.toList());
     }
 
-    public static HourlyTimeWindow toCoreHourlyTimeWindow(com.netflix.titus.grpc.protogen.TimeWindow.HourlyTimeWindow grpcHourlyTimeWindow) {
+    public static HourlyTimeWindow toCoreHourlyTimeWindow
+            (com.netflix.titus.grpc.protogen.TimeWindow.HourlyTimeWindow grpcHourlyTimeWindow) {
         return HourlyTimeWindow.newBuilder()
                 .withStartHour(grpcHourlyTimeWindow.getStartHour())
                 .withEndHour(grpcHourlyTimeWindow.getEndHour())
                 .build();
     }
 
-    public static List<ContainerHealthProvider> toCoreContainerHealthProviders(List<com.netflix.titus.grpc.protogen.ContainerHealthProvider> grpcContainerHealthProviders) {
+    public static List<ContainerHealthProvider> toCoreContainerHealthProviders
+            (List<com.netflix.titus.grpc.protogen.ContainerHealthProvider> grpcContainerHealthProviders) {
         return grpcContainerHealthProviders.stream().map(GrpcJobManagementModelConverters::toCoreHourlyTimeWindow).collect(Collectors.toList());
     }
 
-    public static ContainerHealthProvider toCoreHourlyTimeWindow(com.netflix.titus.grpc.protogen.ContainerHealthProvider grpcContainerHealthProvider) {
+    public static ContainerHealthProvider toCoreHourlyTimeWindow
+            (com.netflix.titus.grpc.protogen.ContainerHealthProvider grpcContainerHealthProvider) {
         return ContainerHealthProvider.newBuilder()
                 .withName(grpcContainerHealthProvider.getName())
                 .withAttributes(grpcContainerHealthProvider.getAttributesMap())
                 .build();
     }
 
-    public static JobDescriptor.JobDescriptorExt toCoreJobExtensions(com.netflix.titus.grpc.protogen.JobDescriptor grpcJobDescriptor) {
+    public static JobDescriptor.JobDescriptorExt toCoreJobExtensions(com.netflix.titus.grpc.protogen.JobDescriptor
+                                                                             grpcJobDescriptor) {
```

Comment:
This formatting looks strange. Is it auto-formatter? We should use common template.

Reply:
Afaik I'm using the shared codestyle template.
Not sure, the autoformat doesn't seem to care about the line len or anything. I'll remove this noise.